theano/tensor/nnet/abstract_conv.py

```diff
@@ -286,6 +294,9 @@ def get_conv_gradinputs_shape(kernel_shape, top_shape,
 
     if filter_dilation is None:
         filter_dilation = np.ones(len(subsample), dtype='int')
+    if num_groups > 1:
+        assert len(subsample) == 2
+        nkern = nkern * num_groups
```

Comment:
I don't get this. If the forward op don't have the output shape changed, why the grads ops would compute different shapes?